limbo/fakeserver.py

```diff
@@ -41,16 +48,11 @@ def __init__(self, server=None, users=None, events=None, bots=None, botname="tes
             "4": User("replbot", 4, "", 0),
         }
 
-        self.bots = bots if bots else {
-            "1": Bot("1", "otherbot", [], False)
-        }
-
+        self.bots = bots if bots else {"1": Bot("1", "otherbot", [], False)}
```

Comment:
I'm not in love with this change. It obviously fits on one line, but I'm not sure it helps readability.

Reply:
in the small, I agree with you; some of the changes definitely are readability negatives.

However I think having a standard format is a big win in general, so I'm willing to take some small negative changes for the larger overall win